readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -132,6 +132,15 @@ if 'html_context' in globals():
 else:
     html_context = context
 
+# Add custom RTD templates
```

Comment:
We were talking to move this code into the `extra_context` block and use it in the corporate site.